src/room.js

```diff
@@ -0,0 +1,36 @@
+class Room {
+
+  constructor (roomname) {
+    this.roomname = roomname;
+    this.endpoints = {};
+  }
+  addEndpoint (endpointId) {
+    this.endpoints[endpointId] = {};
```

Comment:
I still think it would be better to have an the `endpointId` string being created by this `addEndpoint` method. That way, the apps themselves aren't left to declare an `endpointId` in whatever form they choose. @njsfield @esraajb Do you disagree?


See my last comment:

the `addEndpoint` method should create a new endpointId, like I have done [here](https://github.com/foundersandcoders/Live-Peers/pull/56/files#diff-c8a2bf410c4a6e6dcf012df0da242442R10):
```javascript
addEndpoint () {
  let endpointId = // function to create a random string
  this.endpoints[endpointId] = {};
}
```